cdap-ranger/cdap-ranger-binding/src/main/java/co/cask/cdap/security/authorization/ranger/binding/RangerAuthorizer.java

```diff
@@ -126,17 +126,17 @@ public void enforce(EntityId entityId, Principal principal, Set<Action> set) thr
 
   @Override
   public void grant(Authorizable authorizable, Principal principal, Set<Action> set) throws Exception {
-    LOG.warn("Grant operation not supported by Ranger for CDAP");
+    LOG.warn("Please use Ranger Admin UI to grant privileges.");
```

Comment:
Oh - I missed it earlier. This should also throw `UnsupportedOperationException`, right? Otherwise how will a user who uses this from CLI know about this?